packages/lab/src/dropdown-next/DropdownNext.tsx

```diff
@@ -205,8 +205,8 @@ export const DropdownNext = forwardRef(function DropdownNext(
         role="combobox"
         aria-haspopup="listbox"
         aria-expanded={open}
-        aria-activedescendant={disabled ? undefined : activeDescendant}
-        tabIndex={disabled ? -1 : 0}
+        aria-activedescendant={activeDescendant}
+        tabIndex={disabled || readOnly ? -1 : 0}
```

Comment:
dont think we should be adding back the focused on readonly here based on the new issue?

Reply:
the latest changes will be done in a separate PR meant for design updates only :)

Reply:
@libertymayc  here you go https://github.com/jpmorganchase/salt-ds/pull/2186

Reply:
you would still need to be able to tab to it if focusable when readonly right? or im a bit confused why its being added here if its then going to be removed

Reply:
the other PR has been merged into this, you'd see that it's been removed (had it in a separate PR earlier in case this PR gets merged first, but no longer an issue anymore)